IANotificationsTestProject/In App Notifications/IANotificationPresenter.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+//IANotificationPresenter is the class that will be most frequently interacted with in SN.
+//This class will contain a queue of notifications, present them on screen, and manage animations
+//This class will also manage listening to keyboard changes and rotation changes, and manage pausing dismissal if user interacts with notification
+
+class IANotificationPresenter {
```

Comment:
Solid concept!!!

1. Only make `public` properties that should be accessed from the outside
2. Unless you're working on Framework code, you don't need to use `public`
3. Magic numbers (watch out for those!) 🔥 
4. Avoid Force Unwraps please 😄 
5. Aaand a really nipticky comment: `//Text` > `// Text` (space, for readability!)